asterisk/rootfs/usr/share/tempio/pjsip_default.conf.gtpl

```diff
@@ -0,0 +1,55 @@
+; DON'T MODIFY THIS FILE, IT GET'S OVERWRITTEN!
+; IF YOU REALLY WANT TO CHANGE sipjs-phone OR my-codecs, YOU CAN ADD THAT IN pjsip_custom.conf OR EVEN pjsip.conf
+
+; For additional parameters go to https://wiki.asterisk.org/wiki/display/AST/Asterisk+18+Configuration_res_pjsip
+
+{{ if .auto_add }}
+{{ $secret := .auto_add_secret }}
+
+; Common AUTH parameters (same username and same secret for all auto-generated endpoints)
+[sipjs-phone-auth]
+type=auth
+auth_type=userpass
+username=sipjs
```

Comment:
Maybe change the username to the person name? Like the callerid.

Reply:
Makes sense to better distinguish between them.

BTW: where does the person name come from? I have added user account (which has a person associated) in HA with only my first name. However, the callerid also contains my lastname. Never entered it in HA.

Reply:
@nanosonde it comes from HA Persons API (how your last name is there, Idk).